src/api/app/views/webui2/webui/project/side_links/_requests.html.haml

```diff
@@ -1,12 +1,17 @@
 %li
   :ruby
-    if requests.size == 1
+    if incoming_requests.size == 1 || outgoing_requests.size == 1
```

Comment:
what happen if both are 1? I mean that you could have 2 requests.

why not just keeping using `requests` here?

Reply:
unless you are intentionally changing the logic, You should use either ```requests.size``` or ```incoming_requests.size + outgoing_requests.size  == 1```

Reply:
`incoming_requests.size + outgoing_requests.size == 1` could also be wrong, as the same one could be in both.

Reply:
then you recommended leave requests.size == 1 ?

Reply:
> then you recommended leave requests.size == 1 ?

yes